stdlib/2/user.pyi

```diff
@@ -0,0 +1,10 @@
+# Stubs for user (Python 2)
+
+# Docs: https://docs.python.org/2/library/user.html
+# Source: https://hg.python.org/cpython/file/2.7/Lib/user.py
+from typing import BinaryIO, Text, Union
+
+
+home: str
+pythonrc: Union[bytes, Text]
```

Comment:
Well, in this case all the arguments are bytes, so the return will also be bytes.